cmd/grpc-contract/main.go

```diff
@@ -91,46 +91,73 @@ func main() {
 			os.Exit(-1)
 		}
 
-		contract := impl.NewContract(pack, pbPackage, util.ToCamelCase(goType), append(pbFiles, file))
+		contract := impl.NewContract(pack, pbPackage,
+			util.ToCamelCase(goType),
+			[]string{
+				file,
+				path.Join(pbPath, goType+".pb.go"),
+				path.Join(pbPath, "messages.pb.go"),
+			})
+
+		// find the corresponding server interface
+		f, ok := pbFilesMap[contract.Sources[1]]
+		if !ok {
+			fmt.Printf("Failed to corresponding source file for service %v\n", goType)
+			os.Exit(-1)
+		}
+		var serverInterface *parser.GoInterface
+		for _, i := range f.Interfaces {
+			if contract.IsServerInterface(i.Name) {
+				serverInterface = i
+				break
+			}
+		}
+		if serverInterface == nil {
+			fmt.Printf("Failed to corresponding server interface for service %v\n", goType)
+			os.Exit(-1)
+		}
+
+		// find the corresponding server interface
+		f, ok = pbFilesMap[contract.Sources[2]]
+		if !ok {
+			fmt.Printf("Failed to messages source file for service %v\n", goType)
```

Comment:
Failed to find corresponding server interface?

Reply:
OK